rst/cartridge_admin.rst

```diff
@@ -47,6 +47,95 @@ You can do this via the Web interface which is available at
 ``http://<instance_hostname>:<instance_http_port>``
 (in this example, ``http://localhost:8080``).
 
+(Optional) You can bootstrap cluster from existing cluster (*original* cluster in example)
```

Comment:
For better structure, I would add this part under a separate heading. Something like *Bootstrapping from an existing cluster*.